rest/test/db/dbFormattingRules_spec.js

```diff
@@ -94,25 +94,12 @@ describe('db formatting rules', () => {
 		expect(result).to.equal('catapult');
 	});
 
-	describe('can format uint type', () => {
-		const testCases = [
-			{ name: 'uint value 0 (min)', value: new Int32(0), formated: 0 },
-			{ name: 'uint8 value 255', value: new Int32(255), formated: 255 },
-			{ name: 'uint16 value 65535', value: new Int32(65535), formated: 65535 },
-			{ name: 'uint32 value 4294967295 (max)', value: new Int32(-1), formated: 4294967295 },
-			{ name: 'uint32 value 2147483647', value: new Int32(2147483647), formated: 2147483647 },
-			{ name: 'uint32 value 2147483648', value: new Int32(-2147483648), formated: 2147483648 }
-		];
-
-		testCases.forEach(testCase => {
-			it(testCase.name, () => {
-				// Arrange + Act:
-				const result = formattingRules[ModelType.uint](testCase.value);
+	it('can format uint8 type', () => {
```

Comment:
probably missing more test cases? not sure if this represents all the possible conversion issues (ie. max uint8 value is 255)